### ISSUE TITLE
Nagios periodically stops updating

### ISSUE THREAD

**Issue by starchy** [QuantumObject/docker-nagios#17]
The instance I have running shows the last service check run at 09:50, and the current system time as 13:42. The last write to nagios.log was also at 09:50:

> $ tail nagios.log
> [1506097794] wproc:   early_timeout=1; exited_ok=0; wait_status=0; error_code=62;
> [1506097794] Warning: Check of host 'printer-four-copier.eff.org' timed out after 30.01 seconds
> [1506097794] HOST ALERT: printer-four-copier.eff.org;DOWN;SOFT;1;(Host check timed out after 30.01 seconds)
> [1506097794] wproc: Core Worker 295: job 58424 (pid=11029): Dormant child reaped
> [1506097812] HOST ALERT: hostname1.example.org;UP;SOFT;2;PING OK - Packet loss = 0%, RTA = 0.92 ms
> [1506098415] SERVICE ALERT: hostname1.example.org;PING;CRITICAL;SOFT;1;CRITICAL - Plugin timed out
> [1506098468] SERVICE ALERT: hostname1.example.org;PING;OK;SOFT;2;PING OK - Packet loss = 0%, RTA = 4.11 ms
> [1506099054] wproc: Core Worker 296: job 58664 (pid=17692) timed out. Killing it
> [1506099054] wproc: CHECK job 58664 from worker Core Worker 296 timed out after 30.01s
> [1506099054] wproc:   host=hostname1.example.org; service=(null);

> $ docker-compose logs
> Attaching to nagios_nginx_1, nagios_nagios_1
> nagios_1  | *** Running: /etc/my_init.d/startup.sh
> nagios_1  | Starting pre-service scritps in /etc/my_init.d
> nagios_1  | starting rc.local scritps
> nagios_1  | *** Running: /etc/rc.local
> nagios_1  | Booting runit daemon...
> nagios_1  | Process runsvdir running with PID 217
> nagios_1  | Starting pre-service scritps in /etc/my_init.d
> nagios_1  | *** Running: /etc/my_init.d/startup.sh
> nagios_1  | already configured
> nagios_1  | starting rc.local scritps
> nagios_1  | *** Running: /etc/rc.local
> nagios_1  | Booting runit daemon...
> nagios_1  | Process runsvdir running with PID 276
> $ docker-compose exec nagios ps auxww
> USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
> root         1  0.0  0.0  18128  2636 ?        Ss   Sep18   0:00 /bin/bash /sbin/my_init
> root       177  0.0  0.0  65408  2148 ?        Ss   Sep18   0:02 /usr/lib/postfix/sbin/master -w
> postfix    179  0.0  0.0  67640  2088 ?        S    Sep18   0:01 qmgr -l -t unix -u
> root       276  4.6  0.0   4396  1036 ?        S    Sep18 248:06 /usr/bin/runsvdir -P /etc/service log:./nagios/ sh: logwatcher: command not found svlogd: warning: processor failed, restart: /var/log/nagios/ 
> root       280  0.1  0.0   4244  1104 ?        Ss   Sep18   7:45 runsv apache2
> root       281  0.0  0.0   4244   728 ?        Ss   Sep18   0:00 runsv nagios
> root       282  0.0  0.0   4244   608 ?        Ss   Sep18   0:00 runsv cron
> root       283  0.0  0.0  26068  1916 ?        S    Sep18   0:00 /usr/sbin/cron -f
> root       284  6.8  0.0   4388  1244 ?        S    Sep18 365:34 svlogd -tt /var/log/nagios/
> nagios     285  0.1  0.0  75572  3360 ?        S    Sep18   8:46 /usr/local/nagios/bin/nagios /usr/local/nagios/etc/nagios.cfg
> root       288  0.0  0.0  18084  2552 ?        S    Sep18   0:00 /bin/sh /usr/sbin/apache2ctl -D FOREGROUND
> root       290  0.0  0.0 288556  2528 ?        S    Sep18   0:13 /usr/sbin/apache2 -D FOREGROUND
> nagios     291  0.0  0.0  35436  1764 ?        S    Sep18   1:01 /usr/local/nagios/bin/nagios --worker /usr/local/nagios/var/rw/nagios.qh
> nagios     292  0.0  0.0  35432  1804 ?        S    Sep18   1:00 /usr/local/nagios/bin/nagios --worker /usr/local/nagios/var/rw/nagios.qh
> nagios     293  0.0  0.0  35432  1804 ?        S    Sep18   1:01 /usr/local/nagios/bin/nagios --worker /usr/local/nagios/var/rw/nagios.qh
> nagios     294  0.0  0.0  35432  1808 ?        S    Sep18   1:00 /usr/local/nagios/bin/nagios --worker /usr/local/nagios/var/rw/nagios.qh
> nagios     295  0.0  0.0  35456  1676 ?        S    Sep18   1:00 /usr/local/nagios/bin/nagios --worker /usr/local/nagios/var/rw/nagios.qh
> nagios     296  0.0  0.0  35456  1680 ?        S    Sep18   1:00 /usr/local/nagios/bin/nagios --worker /usr/local/nagios/var/rw/nagios.qh
> www-data   297  0.0  0.0 289056  6088 ?        S    Sep18   0:04 /usr/sbin/apache2 -D FOREGROUND
> www-data   298  0.0  0.0 289064  6052 ?        S    Sep18   0:04 /usr/sbin/apache2 -D FOREGROUND
> www-data   299  0.0  0.0 289064  6216 ?        S    Sep18   0:04 /usr/sbin/apache2 -D FOREGROUND
> www-data   300  0.0  0.0 289056  6084 ?        S    Sep18   0:04 /usr/sbin/apache2 -D FOREGROUND
> www-data   301  0.0  0.0 289072  6124 ?        S    Sep18   0:04 /usr/sbin/apache2 -D FOREGROUND
> nagios     302  0.0  0.0  74812  1080 ?        S    Sep18   0:23 /usr/local/nagios/bin/nagios /usr/local/nagios/etc/nagios.cfg
> postfix   7879  0.0  0.0  67476  4376 ?        S    13:38   0:00 pickup -l -t unix -u -c
> www-data  9108  0.0  0.0 289056  6692 ?        S    Sep21   0:00 /usr/sbin/apache2 -D FOREGROUND
> www-data 10493  0.0  0.0 289056  6248 ?        S    Sep18   0:04 /usr/sbin/apache2 -D FOREGROUND
> root     27963  0.0  0.0  34424  2904 ?        Rs+  13:47   0:00 ps auxww

The instance will run for weeks or months without issue, but since there's no alerting to the fact that the checks stop running, this can be a problem.
